gcp/appengine/frontend3/src/templates/home.html

```diff
@@ -134,7 +134,7 @@ <h2 class="heading">Use the API</h2>
       <div class="code-card-container mdc-layout-grid__inner">
         <div class="code-card mdc-layout-grid__cell--span-6 mdc-layout-grid__cell--span-12-tablet">
           <h3 class="code-card-title">Query by commit hash</h3>
-          <pre class="code-card-content" id="example-commit-hash">curl -X POST -d \
+          <pre class="code-card-content" id="example-commit-hash">curl POST -d \
```

Comment:
Looks like there was some de-X-ing here without the next word

```suggestion
          <pre class="code-card-content" id="example-commit-hash">curl -d \
```